ngx-fudis/projects/ngx-fudis/src/lib/components/breadcrumbs/breadcrumbs.component.html

```diff
@@ -0,0 +1,11 @@
+<nav aria-label="Breadcrumb navigation" class="fudis-breadcrumbs" *ngIf="links.length > 0">
+	<ng-container *ngFor="let link of links; let last = last">
+		<fudis-link
+			[disabled]="last"
+			[color]="last ? 'default' : 'primary'"
+			[routerLinkUrl]="link.url"
+			*ngIf="link.url"
+			[linkTitle]="link.label"></fudis-link>
+		<span *ngIf="!last" class="fudis-separator">/</span>
```

Comment:
The current design for breadcrumb component is as follows:

<img width="655" alt="Screenshot 2023-08-30 at 12 58 14" src="https://github.com/funidata/fudis/assets/92301174/b2faf916-5ad0-4b82-b310-885d6e7df8c8">

Change the separator span to `fudis-icon` with chevron.